Client/UserResearch/NewTabUserResearch.swift

```diff
@@ -33,6 +34,14 @@ class NewTabUserResearch {
             return value
         }
     }
+    var hasEnrolled: Bool {
```

Comment:
after discussing with data science we only want to send enrollment event once. We save it to user defaults and check before sending additional enrollment event to our internal telemetry.